ci/jenkins/pipelines/prs/skuba-update-acceptance.Jenkinsfile

```diff
@@ -51,7 +51,7 @@ pipeline {
         } } }
 
         stage('Setting GitHub in-progress status') { steps {
-            sh(script: "${PR_MANAGER} update-pr-status ${GIT_COMMIT} ${PR_CONTEXT} 'pending'", label: "Sending pending status")
+            sh(script: "${PR_MANAGER} update-pr-status ${PR_CONTEXT} 'pending'", label: "Sending pending status")
         } }
 
         stage('Git Clone') { steps {
```

Comment:
Probably remove?

Reply:
Not related to this PR, will do in a follow up PR.

Reply:
Thought that was the reason for the CI failure... ok, if its not.